src/main.go

```diff
@@ -15,7 +16,20 @@ func main() {
 
 	startTime := time.Now()
 
-	configStruct, err := config.LoadConfig("config.json")
+	// define a flag for config file
+	configFile := flag.String("config", "config.json", "Path to configuration file")
```

Comment:
maybe it makes sense to make the default config value: "default-config.json". That way a user can use a config named "config,json". 

If there is no config passed in - then the program should use "default-config.json"